curriculum/challenges/english/25-front-end-development/lab-fortune-teller/66c06d618d075c7f7f1b890a.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 66c06d618d075c7f7f1b890a
+title: Build a Fortune Teller App
+challengeType: 14
+dashedName: build-a-fortune-teller-app
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should initialize the five variables `fortune1`, `fortune2`, `fortune3`, `fortune4`, and `fortune5` with a string value of your choice. You can use the below options if you like:
+    
+- `"Your cat will look very cuddly today."`.
+- `"The weather will be nice tomorrow."`.
+- `"Be cautious of your new neighbours."`.
+- `"You will find a new hobby soon."`.
+- `"It would be wise to avoid the colour red today."`.
+  
+1. You should select a random number between 1 and 5, inclusive, and assign it to the variable `randomNumber`.
+You should use an `if/else if` statement to print a different fortune based on the value of `randomNumber`:
+
+- If `randomNumber` is 1, print the first fortune.
+- If `randomNumber` is 2, print the second fortune.
+- If `randomNumber` is 3, print the third fortune.
+- If `randomNumber` is 4, print the fourth fortune.
+- If `randomNumber` is 5, print the fifth fortune.
+
+1. You should log the fortune to the console.
+   
+# --hints--
+
+You should initialize `fortune1` with a string value.
+
+```js
+assert.isNotNull(fortune1);
+assert.isString(fortune1);
+```
+
+You should initialize `fortune2` with the a string value.
+
+```js
+assert.isNotNull(fortune2);
+assert.isString(fortune2);
+```
+
+You should initialize `fortune3` with a string value.
+
+```js
+assert.isNotNull(fortune3);
+assert.isString(fortune3);
+```
+
+You should initialize `fortune4` with a string value.
+
+```js
+assert.isNotNull(fortune4);
+assert.isString(fortune4);
+```
+
+You should initialize `fortune5` with a string value.
+
+```js
+assert.isNotNull(fortune5);
+assert.isString(fortune5);
+```
+
+You should generate a random number between 1 and 5, inclusive, and assign it to the variable `randomNumber`.
+
+```js
+assert.isAtLeast(randomNumber, 1);
+assert.isAtMost(randomNumber, 5);
```

Comment:
I would include a regex test for `math.random` or something - so they don't just assign it a number.

Reply:
Regex has been included.